test/schema/smbgTest.js

```diff
@@ -43,5 +43,16 @@ describe('schema/cbg.js', function(){
     helper.expectUnitConversion(goodObject, 'value');
   });
 
+  describe('subType', function(){
+    var withSubType = _.assign({}, goodObject, {subType: ''});
+    helper.okIfAbsent(goodObject, 'subType');
+    helper.expectStringField(withSubType, 'subType');
```

Comment:
I guess I was just expecting to see a test that had the subType as
either "linked" or "manual"

On Sat, Jan 31, 2015 at 6:52 AM, Jana Beck <notifications@github.com> wrote:

> In test/schema/smbgTest.js
> <https://github.com/tidepool-org/jellyfish/pull/48#discussion_r23858678>:
>
> > @@ -43,5 +43,16 @@ describe('schema/cbg.js', function(){
> >      helper.expectUnitConversion(goodObject, 'value');
> >    });
> >
> > +  describe('subType', function(){
> > +    var withSubType = _.assign({}, goodObject, {subType: ''});
> > +    helper.okIfAbsent(goodObject, 'subType');
> > +    helper.expectStringField(withSubType, 'subType');
>
> No, the first one checks if it's OK that the whole field is absent. Note
> that I give it the unmodified goodObject there, not the new withSubType
> copy. Could split it into two tests if you like?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/tidepool-org/jellyfish/pull/48/files#r23858678>.
>